src/app/components/user-profile/user-profile.component.spec.ts

```diff
@@ -19,7 +19,12 @@ describe('UserProfileComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  it('Prov dropdown value should change after select Country ', () => {
+    const form = component.userProfileForm;
+    const countryInput = form.controls.country;
+    const provInput = form.controls.province;
+    countryInput.setValue('Canada');
+    // expect(form.getProv.contain('ON')).toBeTruthy();
```

Comment:
Actually this is a very good start. Here some guidance though : 
- You remove the logic introduce by @edwinyachoui-ibm regarding the 2 dropdowns
![image](https://user-images.githubusercontent.com/64276312/98250731-d283f580-1f45-11eb-85b2-e87c75527c32.png)
This is going to be tricky to test ... whatever you want to test around :)
- According to your test definition you are looking for the actual values of the dropdown `Prov dropdown value should change after select Country` : good. So ask yourself, what should you compare ... to what exactly ?

Reply:
Ye, just realize my version was the previous one which didn't contain dropdown logic, will have it updated.